androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivityMetadata.java

```diff
@@ -278,9 +287,33 @@ private static TrustedWebActivityDisplayMode getDisplayMode(@NonNull Bundle meta
         if ("browser".equals(displayMode)) {
             return new TrustedWebActivityDisplayMode.BrowserMode();
         }
+        
+        if (includeExperimental) {
+            if ("window-controls-overlay".equals(displayMode)) {
+                return new TrustedWebActivityDisplayMode.WindowControlsOverlay();
```

Comment:
Naming this `WindowControlsOverlay` instead of `WindowControlsOverlayMode` was an unfortunate oversight. I believe we can correct it in another alpha release of Androidx browser package.